pkg/cfn/builder/cluster.go

```diff
@@ -0,0 +1,78 @@
+package builder
+
+import (
+	"net"
+
+	cfn "github.com/aws/aws-sdk-go/service/cloudformation"
+	gfn "github.com/awslabs/goformation/cloudformation"
+
+	"github.com/weaveworks/eksctl/pkg/eks/api"
+)
+
+const (
+	cfnOutputClusterCertificateAuthorityData = "CertificateAuthorityData"
+	cfnOutputClusterEndpoint                 = "Endpoint"
+	cfnOutputClusterARN                      = "ARN"
+	cfnOutputClusterStackName                = "ClusterStackName"
+)
+
+type clusterResourceSet struct {
+	rs             *resourceSet
+	spec           *api.ClusterConfig
+	subnets        []*gfn.StringIntrinsic
+	securityGroups []*gfn.StringIntrinsic
+}
+
+func NewClusterResourceSet(spec *api.ClusterConfig) *clusterResourceSet {
+	return &clusterResourceSet{
+		rs:   newResourceSet(),
+		spec: spec,
+	}
+}
+
+func (c *clusterResourceSet) AddAllResources() {
+	c.rs.template.Description = clusterTemplateDescription
+	c.rs.template.Description += clusterTemplateDescriptionDefaultFeatures
+	c.rs.template.Description += templateDescriptionSuffix
+
+	_, globalCIDR, _ := net.ParseCIDR("192.168.0.0/16")
```

Comment:
These CIDR are hardcoded, as that's what we had before, except that it was in YAML...

Reply:
Do you think this will always be hardcoded?

Reply:
No, not at all. These will ultimately get set from a config file (when we have that). I'm less keen on having flags for each of these. They are here for now, I just didn't want to do anything clever about these as part of this PR.